echidna/core/smear.py

```diff
@@ -0,0 +1,224 @@
+import numpy
+import decimal
+import spectra
+
+class Smear(object):
+    """ This class smears the energy and radius of a spectra.
+
+    The class can recieve energy and radius as individual data points or a 
+    1 dimensional numpy array to smear which is then returned. 2d and 3d 
+    arrays with linked energy, radius and time information is yet to be
+    implemented.
+
+    Attributes:
+      _light_yield (float): Number of PMT hits expected for a MeV energy deposit in NHit/MeV
+      _position_resolution (float): Sigma in mm
+    """
+    _light_yield = 200. # NHit per MeV
+    _position_resolution = 100. # mm
+
+    def __init__(self):
+        """ Initialise the Smear class by seeding the random number generator
+        """
+        numpy.random.seed()
+
+    def bin_1d_array(self, array, bins):
+        """ Sorts a 1 dimensional array and bins it
+        
+        Args:
+          array (numpy array): To sort and bin
```

Comment:
If you write this as ``array (:class:`numpy.array`)`` it will format it nicer in the docs